selfdrive/car/mazda/interface.py

```diff
@@ -10,6 +10,9 @@
 EventName = car.CarEvent.EventName
 
 class CarInterface(CarInterfaceBase):
+  def __init__(self, CP, CarController, CarState):
+    super().__init__(CP, CarController, CarState)
+    self.dp_mazda_ti = Params().get_bool('dp_mazda_ti')
```

Comment:
add params to dp_conf.py `../common/dp_conf.py` and then use can just use dragoncof.

Reply:
Added to dp_conf.py, interfaces.py is fine as it is now right?